packages/auth/src/gql/getPasswordResetSchema.ts

```diff
@@ -45,9 +50,8 @@ export function getPasswordResetSchema<I extends string, S extends string>({
         type: graphql.nonNull(graphql.Boolean),
         args: { [identityField]: graphql.arg({ type: graphql.nonNull(graphql.String) }) },
         async resolve(rootVal, { [identityField]: identity }, context) {
-          const dbItemAPI = context.sudo().db[listKey];
+          const dbItemAPI = (context as ContextWithOnlyStandardLists).sudo().db[listKey];
```

Comment:
```suggestion
          const dbItemAPI = getContextDb(context.sudo(), listKey);
```
And remove the `as ContextWithOnlyStandardLists` types